### PR TITLE
Close window (pipelined-rendering)

### PR DESCRIPTION
**Pull request by DJMcNab**
# Objective

- We didn't have any way to close specific windows

## Solution

- Integrate closing windows properly

Note that this still has some issues with bevy_render - in particular, I'm unsure what to do to/about the corresponding `WindowTextureNode`/`WindowSwapChainNode` when the window is closed.

At the moment, we panic whenever a window is closed, assuming it had anything drawn to it. This is not ideal.

### CODE REVIEW

**Review by TheRawMeatball — APPROVED** [review]

File: crates/bevy_winit/src/lib.rs
Comment (bjorn3) [inline crates/bevy_winit/src/lib.rs:173]: !removed_windows.is_empty() ?

**Comment by DJMcNab** [comment]
Fixing the render work should wait until the new renderer is merged, unless this gets moved over to there.

**Comment by DJMcNab** [comment]
Pipelined has been merged, might re-open later.

**Pull request closed without merging**
